frontend/ui/wikipedia.lua

```diff
@@ -655,6 +655,31 @@ local ext_to_mimetype = {
     woff = "application/font-woff",
 }
 
+-- Display from these Wikipedia should be RTL
```

Comment:
Don't be afraid to use multiline comments. ;-)

Reply:
Remind me what are these and how they differ from consecutive `-- blah` ?! :)

Reply:
I just mean 
```
--[[--lua
Bah blah.

Blah.
--]]
```

Reply:
Well, I find these uglier :)